web/app/js/components/TapQueryForm.jsx

```diff
@@ -106,13 +123,18 @@ export default class TapQueryForm extends React.Component {
     };
   }
 
+  handleUrlUpdate = (name, formVal) => {
```

Comment:
Maybe we should add a comment here explaining that we're calling the onChange that's added by `react-url-query`, since it's not obvious which prop handler this is calling?